backend/Generator/main.py

```
@@ -45,54 +83,75 @@ def set_seed(self, seed):
-        if len(keyword_sentence_mapping.keys()) == 0:
-            return final_output
-        else:
+        if len(keyword_sentence_mapping) > 0:
             try:
                 generated_questions = generate_multiple_choice_questions(keyword_sentence_mapping, self.device, self.tokenizer, self.model, self.s2v, self.normalized_levenshtein)
-            except:
-                return final_output
-
-            end_time = time.time()
+                final_output["questions"] = generated_questions.get("questions", [])
+            except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)